pkg/apis/flows/v1/parallel_lifecycle.go

```diff
@@ -42,6 +42,10 @@ const (
 	// ParallelConditionAddressable has status true when this Parallel meets
 	// the Addressable contract and has a non-empty hostname.
 	ParallelConditionAddressable apis.ConditionType = "Addressable"
+
+	// ParallelConditionEventPoliciesReady has status True when applying EventPolicies for this
```

Comment:
It will become True too, if no policy applies

Reply:
sure, done PTAL!